diff --git a/dist/client/with-control.js b/dist/client/with-control.js
index 38ba81d96e538143288f4540418c95141938bbff..3a43a1b9cf8969b098caa0cba151de931a9fa6ee 100644
--- a/dist/client/with-control.js
+++ b/dist/client/with-control.js
@@ -56,6 +56,7 @@ function WithControl({ presets, displayName, Component }) {
 					field: preset.controls,
 					fieldName: [],
 					name: t("Props"),
+					collapsible: false,
 					className: "max-h-[600px] overflow-auto"
 				})
 			]
