diff --git a/Users/tyler.joo/Projects/seed-design/node_modules/@radix-ui/react-dialog/.bun-tag-d722d149ad58556a b/.bun-tag-d722d149ad58556a
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/index.js b/dist/index.js
index ce328f1bc67710ca5c3bf303d552ba7be0b87a2e..1d5bd1814a860de120dd758fe91ae07f5c2b6b45 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -65,7 +65,7 @@ var import_react_portal = require("@radix-ui/react-portal");
 var import_react_presence = require("@radix-ui/react-presence");
 var import_react_primitive = require("@radix-ui/react-primitive");
 var import_react_focus_guards = require("@radix-ui/react-focus-guards");
-var import_react_remove_scroll = require("react-remove-scroll");
+// var import_react_remove_scroll = require("react-remove-scroll");
 var import_aria_hidden = require("aria-hidden");
 var import_react_slot = require("@radix-ui/react-slot");
 var import_jsx_runtime = require("react/jsx-runtime");
@@ -149,11 +149,22 @@ var DialogOverlay = React.forwardRef(
   }
 );
 DialogOverlay.displayName = OVERLAY_NAME;
-var Slot = (0, import_react_slot.createSlot)("DialogOverlay.RemoveScroll");
+// var Slot = (0, import_react_slot.createSlot)("DialogOverlay.RemoveScroll");
 var DialogOverlayImpl = React.forwardRef(
   (props, forwardedRef) => {
     const { __scopeDialog, ...overlayProps } = props;
     const context = useDialogContext(OVERLAY_NAME, __scopeDialog);
+
+    return (0, import_jsx_runtime.jsx)(
+        import_react_primitive.Primitive.div,
+        {
+          "data-state": getState(context.open),
+          ...overlayProps,
+          ref: forwardedRef,
+          style: { pointerEvents: "auto", ...overlayProps.style }
+        }
+      );
+
     return (
       // Make sure `Content` is scrollable even when it doesn't live inside `RemoveScroll`
       // ie. when `Overlay` and `Content` are siblings
diff --git a/dist/index.mjs b/dist/index.mjs
index 13581033b5a5635796d4cf492f0828e853d011c4..55ce614f95b06f7d0233b331ffcf374e99f55b3b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -13,9 +13,9 @@ import { Portal as PortalPrimitive } from "@radix-ui/react-portal";
 import { Presence } from "@radix-ui/react-presence";
 import { Primitive } from "@radix-ui/react-primitive";
 import { useFocusGuards } from "@radix-ui/react-focus-guards";
-import { RemoveScroll } from "react-remove-scroll";
+// import { RemoveScroll } from "react-remove-scroll";
 import { hideOthers } from "aria-hidden";
-import { createSlot } from "@radix-ui/react-slot";
+// import { createSlot } from "@radix-ui/react-slot";
 import { Fragment, jsx, jsxs } from "react/jsx-runtime";
 var DIALOG_NAME = "Dialog";
 var [createDialogContext, createDialogScope] = createContextScope(DIALOG_NAME);
@@ -97,11 +97,22 @@ var DialogOverlay = React.forwardRef(
   }
 );
 DialogOverlay.displayName = OVERLAY_NAME;
-var Slot = createSlot("DialogOverlay.RemoveScroll");
+// var Slot = createSlot("DialogOverlay.RemoveScroll");
 var DialogOverlayImpl = React.forwardRef(
   (props, forwardedRef) => {
     const { __scopeDialog, ...overlayProps } = props;
     const context = useDialogContext(OVERLAY_NAME, __scopeDialog);
+
+    return jsx(
+        Primitive.div,
+        {
+          "data-state": getState(context.open),
+          ...overlayProps,
+          ref: forwardedRef,
+          style: { pointerEvents: "auto", ...overlayProps.style }
+        }
+      );
+
     return (
       // Make sure `Content` is scrollable even when it doesn't live inside `RemoveScroll`
       // ie. when `Overlay` and `Content` are siblings
